Openshift-EE/pipelines/OpenEBS-base/stages/4-chaos/1PP5-app-kill/app-kill

```diff
@@ -50,7 +54,40 @@ python3 Openshift-EE/utils/result/result_update.py $job_id 1PP5 4-chaos "Induce
 exit 1;
 fi
 
-echo "*******Performing Application POd Chaos**********"
+################
+# LitmusBook 2 #
+################
+
+echo "*******Deploying Busybox Loadgen****"
+
+run_id="app-kill";test_name=$(bash Openshift-EE/utils/generate_test_name testcase=busybox-liveness metadata=${run_id})
+echo $test_name
+
+cd litmus
+cp apps/busybox/liveness/run_litmus_test.yml busybox_loadgen_app_kill.yml
+
+sed -i -e 's/value: app-busybox-ns/value: app-kill/g' \
+-e 's/app=busybox-sts/app=app-kill/g' \
+-e 's/liveness: litmus-busybox-liveness/liveness: busybox-liveness-appkill/g' busybox_loadgen_app_kill.yml
+
+cat busybox_loadgen_app_kill.yml
+
+bash ../Openshift-EE/utils/litmus_job_runner label='liveness:busybox-liveness-appkill' job=busybox_loadgen_app_kill.yml
+cd ..
+bash Openshift-EE/utils/dump_cluster_state;
+bash Openshift-EE/utils/event_updater jobname:app-kill $test_name jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
+
+if [ "$?" != "0" ]; then
+python3 Openshift-EE/utils/result/result_update.py $job_id 1PP5 4-chaos "Induce application pod failure and check if it is recovered successfully" Fail $pipeline_id "$current_time" $commit_id $gittoken
```

Comment:
@shashank855 Instead of  passing the test result status as `fail` can you get the testresult status from litmusresult
for ex : 
`testResult=$(kubectl get litmusresult ${test_name} --no-headers -o custom-columns=:spec.testStatus.result)`

Reply:
@nsathyaseelan If test fails command returns non zero rc. Then why to perform one extra execution of command ?